addons/mail/static/src/core/common/call_notification_message.js

```diff
@@ -0,0 +1,37 @@
+import { Component, useState } from "@odoo/owl";
+import { formatDuration } from "@web/core/l10n/dates";
+import { _t } from "@web/core/l10n/translation";
+import { useService } from "@web/core/utils/hooks";
+
+export class CallNotificationMessage extends Component {
+    static template = "mail.CallNotificationMessage";
+    static props = ["author", "messageDate", "dateStart", "dateEnd?"];
+
+    setup() {
+        super.setup();
+        this.store = useState(useService("mail.store"));
+    }
+
+    get callInformation() {
+        if (this.props.dateStart.equals(this.props.dateEnd)) {
+            return _t("started a call.");
+        }
+        return _t("started a call that lasted %(duration)s.", {
+            duration: this.callDuration,
+        });
+    }
+
+    get callDuration() {
+        const diff = this.props.dateEnd.diff(this.props.dateStart, ["hours", "minutes", "seconds"]);
+        // const diff = Duration.fromObject({ hours: 1, minutes: 0, seconds: 15 })
+        const { hours, minutes } = diff.toObject();
+        if (!hours && !minutes) {
+            return _t("a few seconds");
+        } else if (!hours && minutes === 1) {
+            return _t("a minute");
+        } else if (hours === 1 && !minutes) {
+            return _t("an hour");
+        }
+        return formatDuration(diff.as("seconds"), true);
```

Comment:
Don't break sentence, this doesn't work in other languages.

You should have a global pattern: "Call laster %(duration)s".

I would drop the name of caller in the notification summary when call ended: this is not that useful and it's more important to keep notification footprint small as to not be distracting compared to message content. Also the duration should use the short version, e.g. `1hr.`, `5min.`, etc, again to keep the notification short.

I don't mind to keep caller name in the notification that tells who initiated the call, but not in the summary.